integrations/drip/drip_test.go

```diff
@@ -148,6 +148,73 @@ func TestTrack(t *testing.T) {
 	}
 }
 
+func TestPageErrorWhenNoEmailPresent(t *testing.T) {
+	os.Setenv("DRIP_API_TOKEN", "123")
+	os.Setenv("DRIP_ACCOUNT_ID", "321")
+	api := &APIMock{Url: "http://www.example.com"}
+	drip := Drip{}
+	drip.api = api
+	page := integrations.Page{
+		Name:       "account.created",
+		UserID:     "123",
+		Properties: map[string]interface{}{},
+		Timestamp:  1234567,
+		ReceivedAt: 65,
+	}
+	err := drip.Page(page)
+	if err == nil {
+		t.Error("Expected error when no email given")
+	}
+}
+
+func TestPage(t *testing.T) {
+	os.Setenv("DRIP_API_TOKEN", "123")
+	os.Setenv("DRIP_ACCOUNT_ID", "321")
+	drip := Drip{}
+	api := APIMock{Url: "http://www.example.com"}
+	drip.api = &api
+	page := integrations.Page{
```

Comment:
Could you add a URL to this `Page`? It's the core and it's not tested, we need to make sure we're sending it to Drip.